cdap-common/bin/functions.sh

```diff
@@ -659,15 +659,40 @@ cdap_check_and_set_classpath_for_dev_environment() {
 # returns "distributed" or "sdk" based on current CDAP_HOME
 #
 cdap_context() {
-  local readonly __context
-  if [[ -e ${CDAP_HOME}/lib/co.cask.cdap.cdap-standalone-@@project.version@@.jar ]]; then
+  local readonly __context __version=$(cdap_version}
```

Comment:
Good catch. Missed that when I updated this branch after merging the context stuff.